src/libs/OptionsListUtils.ts

```diff
@@ -1931,8 +1939,18 @@ function getOptions(
             return;
         }
 
-        if ((!accountIDs || accountIDs.length === 0) && !isChatRoom) {
-            return;
+        // Reports that aren't searchable via participant won't have any participants returned with SearchForReports
+        // so we don't need to check if their participants exist in order for them to qualify as a valid option.
+        if (isSearchingForReports && !ReportUtils.isSearchableViaParticipants(report)) {
+            return option;
+        }
+
+        if (!accountIDs || accountIDs.length === 0) {
+            if (!isSearchingForReports) {
```

Comment:
Can we add a comment that generally explains why we skip reports when:

- they have no participants
- we are not searching for reports
- it's not a chat room

Reply:
We don't really need to change anything with this logic because I made it so that all chat rooms are just the reports that we don't want searchable via participant.